scrabble.js

```diff
@@ -1,14 +1,140 @@
 const Scrabble = {
-  score: function(word) {
-    // TODO: implement score
-  }
+  // https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/10-JavaScript/fun-with-functions.md
+  // in the "attaching functions to objects" what's written is inconsistent
+  // with what's here which is inconsistent with what
+  // linter underlines in red.
+
+  // Documentation says it should be highestScoreFrom: function(params){} but
+  // this makes linter angry
+  score(word) {
+    const lowerCaseWord = word.toLowerCase();
+    const points = {
+      one: ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't'],
+      two: ['d', 'g'],
+      three: ['b', 'c', 'm', 'p'],
```

Comment:
While this approach certainly works, it would be more concise (and efficient) to store a mapping from letter to score, rather than the other way around. Something like:

```javascript
// points maps from letter => score
const points = {
  a: 1, b: 3, c: 3, // ...
}
// ...
// loop through letters, add score to total
for (let i = 0; i < lowerCaseWord.length; i++) {
  let value = points[lowerCaseWord[i]];
  if (!value) {
    throw new Error('You must use valid characters');
  }
  total += value;
}
```